TravelAgency-API/src/main/java/cz/muni/fi/travelAgency/DTO/ExcursionCreateDTO.java

```diff
@@ -47,6 +47,11 @@
     @NotNull
     private Duration excursionDuration;
 
+    /**
+     * Pretty print representation of duration.
+     */
+    private String durationString;
```

Comment:
I don't think holding this as an field is a good idea, I would prefer just rename the method initDurationString and make it public